Seeing below and also ci is failing

```
Error: ❌  DCO check failed.

Commits missing the 'Signed-off-by:' footer:
f070dba

🛠 **Quick fix (make ONE signed commit):**
    git fetch origin
    git reset --soft origin/main
    git commit -s -m "<your message>"
    git push --force-with-lease

🔧 **Fix individual commits:**
    git rebase -i origin/main --exec "git commit --amend -s --no-edit"
    git push --force-with-lease

💡  Or edit the commit message in GitHub UI and add:
    Signed-off-by: Your Name <email@example.com>
Error: Process completed with exit code 1.
```

see https://github.com/openai/codex?tab=readme-ov-file#developer-certificate-of-origin-dco.

Could you resolve?

--- a/codex-cli/src/utils/singlepass/context.ts
+++ b/codex-cli/src/utils/singlepass/context.ts
@@ -51,14 +51,15 @@ export function renderTaskContext(taskContext: TaskContext): string {
  * For each file, we embed the content in a CDATA section.
  */
 function renderFilesToXml(files: Array<FileContent>): string {
-  let xmlContent = "<files>";
-  for (const fc of files) {
-    xmlContent += `
+  const fileContents = files
+    .map(
+      (fc) => `
       <file>
         <path>${fc.path}</path>
         <content><![CDATA[${fc.content}]]></content>
-      </file>`;
-  }
-  xmlContent += "\n</files>";
-  return xmlContent;
+      </file>`,
+    )
+    .join("");
+
+  return `<files>\n${fileContents}\n</files>`;
 }

